index.html

```diff
@@ -601,6 +601,180 @@ <h2 class="sg-section-title structure">Forms</h2>
                         </header>
                         <div class="sg-section-content structure">
                             
+
+    
+        <h3 class="section-sub-title">Example Form</h3>
+
+        <form action="" method="post">
+            <div class="error-block form-error">
+                <p class="error">This is an example form error</p>
+            </div>
+
+            
+                <fieldset class="text-fieldset">
+                    <div class="control-group text required">
+                        <label for="field-name" class="control-label">
+                            Text input
+                            <span class="asterisk">*</span>
+                        </label>
+                        <div class="controls">
+                            <input class="text-input" id="field-name" type="text" name="name" placeholder="Text input" aria-required="true">
+                        </div>
+                    </div>
+
+                    <div class="control-group email required">
+                        <label for="field-email" class="control-label">
+                            Email input
+                            <span class="asterisk">*</span>
+                        </label>
+                        <div class="controls">
+                            <input class="email-input" id="field-email" type="email" name="email" placeholder="Email input" aria-required="true">
```

Comment:
Should `name` and `id` be the same? I know they don't technically have to be, but a lot of frameworks etc. often assume them to be and provide that in their default output? 

`id` is mostly used for styling and locating the element in javascript CSS selectors e.g. with jQuery `$("#field-id")`. 

`name` is used as the form field key by the browser when submitting the form, or by frameworks like angular when looking at the form fields.

Not sure if these need to be the same? Maybe there's value in having them different as well?